frontend/gulpfile.js

```diff
@@ -19,6 +19,8 @@ gulp.task('watch', ['serve'], function(){
     gulp.watch(paths.appSrc, ['scripts']);
     gulp.watch(paths.bowerSrc, ['vendors']);
     gulp.watch(paths.index, ['copyAll']);
+    gulp.src("../cisco_logo.png")
```

Comment:
Nice, ar trebui sa functioneze astfel, however, incercam pe cat de mult sa nu punem pathuri hardcodate in cod. 

Creeaza o variabila noua in obiectul paths, unde sa faci store la pathul asta. Astfel, daca pe viitor, o sa avem nevoie de pathul asta in 2 locuri, si vrem sa il modificam in ambele locuri, trebuie updatata doar variabila din obiectul paths.

Reply:
Makes sense, astfel acum ar trebui sa inchid pull request-ul si sa fac altul dupa ce fac commit cu modificarea( sau ar trebui sa fac squash la cele doua commit-uri in unul singur) ?